Assets/Mirror/Runtime/NetworkManagerHUD.cs

```diff
@@ -106,11 +106,11 @@ void StartButtons()
         void StatusLabels()
         {
             // server / client status message
-            if (NetworkServer.active)
+            if (NetworkClient.isConnected)
```

Comment:
shouldn't 'server:active' be shown if networkserver.active?
I don't understand this change, please explain